### PR TITLE
Sh 12 articles

### LINKED ISSUE #12: Create articles pages

### PR DESCRIPTION
**Pull request by SamHollings**
# Description

❓**What**: closes #12 - makes a section of the website for articles/blogs, and added an example currently public.

🧠**Why?**: It's important we can easily blog.

👨‍💻**How?**: Uses mkdocs materials blog plugin.

# Checklist:
Have checked for the following:
- [x] The website still builds correctly, and you can view it using `mkdocs serve`.
- [x] There are no new "warnings" from mkdocs
- [x] Does your page follow the [page template](https://nhsdigital.github.io/rap-community-of-practice/example_RAP_CoP_page/) (or [here in Markdown](https://github.com/NHSDigital/rap-community-of-practice/blob/main/docs/example_RAP_CoP_page.md))? (**need to make a new one specific to NHSE Data Science**)
- [x] Spelling errors
- [x] Consistent capitalization
- [x] Consistent numbers
- [x] Material features incorrectly implemented: search for code blocks and markers (e.g. !!!).
- [x] Code snippets don't work
- [x] Images not working
- [x] Links not working

## Where it was tested

- Github Codespaces - 2-core, 4GB RAM, 32GB hard drive
- devcontainer.json describes further settings

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -33,20 +33,7 @@ To increase the likelihood of your pull request being accepted:
 Run the commands (or follow the MkDocs documentation to locally pip install MkDocs):
```
Comment (amaiaita) [inline CONTRIBUTE.md:33]: wont let me add this comment to line 29, but line 29 still says RAP, we want it to say Data Science website :)

File: mkdocs.yml [hunk 2]
```diff
@@ -11,12 +11,15 @@ plugins:
...
+  - blog:
+      blog_dir: ./articles
+      authors_file: "{blog}/.authors.yml"
 nav:
   - Home: index.md
   - What is Data Science?:
       - ... | what_is_data_science/*.md
   - Articles:
-      - ... | articles/*.md
+      - ... | articles/**.md
```
Comment (amaiaita) [inline mkdocs.yml:22]: just for my own understanding - how come it needs two *s?

Comment (SamHollings) [reply mkdocs.yml:22]: My own mistake - seems redundant. For some reason I though that was needed to make it recursive... but seems not!

File: CONTRIBUTE.md (hunk 1 above)
Comment (SamHollings) [reply CONTRIBUTE.md:33]: I've changed a few other references to NHS Digital and RAP

**Pull request merged**